### PR TITLE
Add support for input_button / Button Helper

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Core PR: https://github.com/home-assistant/core/pull/62008

image

image

image

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
I think you need to add it to `DOMAINS_INPUT_ROW` as well, since otherwise it probably will not work on mobile (instead the more-info would popup when pressing the button).

**Comment by frenck** [comment]
> I think you need to add it to DOMAINS_INPUT_ROW as well, since otherwise it probably will not work on mobile (instead the more-info would popup when pressing the button).

In that case, isn't the `button` entity incorrect in the current state as well?

**Comment by spacegaier** [comment]
> In that case, isn't the button entity incorrect in the current state as well?

Yes, hence this PR I created 😉 https://github.com/home-assistant/frontend/pull/10940

**Comment by frenck** [comment]
Haha alright we push out this change :)

File: src/panels/config/helpers/forms/ha-input_button-form.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+class HaInputButtonForm extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public new?: boolean;
+
+  private _item?: InputButton;
+
+  @state() private _name!: string;
+
+  @state() private _icon!: string;
```
Comment (zsarnett) [inline src/panels/config/helpers/forms/ha-input_button-form.ts:20]:
```suggestion
  @property() public new?: boolean;

  @state() private _name!: string;

  @state() private _icon!: string;

  private _item?: InputButton;
```

Comment (zsarnett) [reply src/panels/config/helpers/forms/ha-input_button-form.ts:20]: Reordering

**Review by zsarnett — DISMISSED** [review]

File: src/panels/config/helpers/forms/ha-input_button-form.ts (hunk 1 above)
Comment (frenck) [reply src/panels/config/helpers/forms/ha-input_button-form.ts:20]: 🤷‍♂️

**Review by zsarnett — APPROVED** [review]

**Comment by frenck** [comment]
Thanks @zsarnett ❤️

**Pull request merged**